src/chi/javascript/components/tooltip.js

```diff
@@ -92,12 +92,11 @@ class Tooltip extends Component {
   _createTooltip () {
     this._tooltipElem = document.createElement('div');
     this._tooltipElem.setAttribute('class', 'chi-tooltip');
-    this._tooltipElem.id = COMPONENT_TYPE +
-      Util.getData(this._elem, COMPONENT_TYPE);
+    this._tooltipElem.id = 'chi-' + COMPONENT_TYPE + '-' + this.componentCounterNo;
```

Comment:
You can use a string template for this kind of things:
```suggestion
    this._tooltipElem.id = `chi-${COMPONENT_TYPE}-${this.componentCounterNo}`;
```